azure-pipelines.yml

```diff
@@ -475,10 +475,11 @@ jobs:
         condition: not(eq(variables['skip-github'], 'TRUE'))
       - bash: |
           set -euo pipefail
+          pr_handler=$(head -1 release/rotation | awk "${print $1}")
```

Comment:
This step of the release process runs on the trigger commit on the master branch, so the `release/rotation` file is the right one. (As opposed to many steps in the release process that checkout the target commit.)